content/en/docs/getting-started/jwk/_index.md

```diff
@@ -22,6 +22,8 @@ Build the code with:
 ```shell
 mvn package
 ```
+A Maven version supporting Java 11 is required. 
```

Comment:
May be give a minimum version number. I have the feeling that it is 3.6.0.

Reply:
I agree, thanks.